tgui/packages/tgui-panel/styles/tgchat/chat-dark.scss

```
@@ -949,16 +957,111 @@ em {
   margin-left: 3em;
 }
 
-.examine_block {
-  background: #1b1c1e;
-  border: 1px solid #a4bad6;
-  margin: 0.5em;
+.tooltip {
+  font-style: italic;
+  border-bottom: 1px dashed;
+}
+
+.fieldset_legend {
+  position: relative;
+  max-width: 95%;
+  font-size: 120%;
+  padding: 0.2em 0.33em;
+  background: #151515; // Chat background color
+  border: 1px solid;
+  border-color: inherit;
+  border-radius: 0.33em;
+  z-index: 1;
```

```suggestion
  z-index: 0;
```
Better make it zero, with 1 black rectangle will be visible if fieldset highlighted